crates/autopilot/src/run_loop.rs

```diff
@@ -310,10 +310,16 @@ impl RunLoop {
                 }
             }
             let solution_uids = solution.order_ids().copied().collect::<HashSet<_>>();
+            let auction_uids = auction.orders.iter().map(|o| o.uid).collect::<HashSet<_>>();
+
             let unsettled_orders: HashSet<_> = solutions
                 .iter()
+                // Report orders that were part of any solution candidate
                 .flat_map(|p| p.solution.order_ids())
+                // but not part of the winning one
                 .filter(|uid| !solution_uids.contains(uid))
+                // yet still part of the auction (filter out jit orders)
```

Comment:
nit: do we want to keep cow amm orders though? if so, we could use `auction.surplus_capturing_jit_order_owners` to detect those.

Reply:
We don't want to track cow amm orders. Specifically on chains with low tx costs (ie gnosis) they tend to trade miniscule amounts that sophisticated solvers may deliberately ignore (e.g. https://gnosisscan.io/tx/0xde46205dab0203bb31c25074e5e7bbd4d2bfdf792d5d4bc9f4c838af407d1f16)

I would also argue since CoW AMMs are passive there is no user waiting for a happy moo or trade notification so it's not so bad if they get delayed (we shouldn't alert on this)